test_unstructured/partition/test_common.py

```diff
@@ -1,3 +1,7 @@
+import datetime as dt
+import io
+import os
+import pathlib
 from dataclasses import dataclass
 from unittest import mock
 
```

Comment:
Only potential change I can think of is adding an integration test to show we can partition a `io.BytesIO` "file" via `file=`. Approving in advance, though.

Reply:
Yeah, good call. I added one for XLSX as a representative example.